.azure-pipelines/templates/jobs/documentation-tests-jobs.yml

```diff
@@ -0,0 +1,48 @@
+jobs:
+  - job: doc_test
+    strategy:
+      matrix:
```

Comment:
Given the short time needed to execute each check, and the fact that it is unlikely that more than 2 packages are modified at the same time, I think it would be relevant to execute the test on all packages in the same job, instead of splitting all of them in parallel.